bc_ccv_rstudio/form.yml

```diff
@@ -11,17 +11,36 @@ form:
   - r_module
   - bc_email_on_started
 attributes:
-
+  bc_account:
+    label: "Condo account"
+    help: "You can leave this blank if you are not using a condo else enter the condo name here for example `cbc-condo`"
   bc_num_slots: 1
+  bc_queue:
+    value: ""
+    label: "Partition"
+    help: "You can leave this blank if you are using the default for your account. Else you can specify the partition you want to run the jobs on in this field. A detailed overview of the partitions on OSCAR can be found in our [docs](https://docs.ccv.brown.edu/oscar/submitting-jobs/slurm) "
   num_cores:
     value: 1
     widget: "number_field"
     help: "Indicate the number of cores you need. The default is 1"
   sh_mem:
     label: "Memory (GB) (optional)"
     widget: "number_field"
-    help: "<small>Amount of memory to allocate</small>"
+    help: "Specify the amount of memory to allocate for the Job in Gigabytes"
     value: 4
+  modules:
+    value: ""
+    help: "You can leave this blank if you are not loading additional modules. Otherwise load mulitple modules by module names seperated by spaces. For example to load mulitple modules use: `abc/12.3  python/2.7` "
```

Comment:
When I run `module load R/4.1.0` Oscar prompts me to load other modules: `module: R: To use: module load gcc/10.2 pcre2/10.35 intel/2020.2 texlive/2018` Are these already loaded by OOD by default? Or do we need some guidance in the docs re: other modules that R might need?